### DISCUSSION TITLE
问题请教：表单验证未通过，但仍然执行了提交逻辑

### DISCUSSION THREAD

**Discussion by xiong-xiao-yang** [layui/layui#1946] [问答]
![表单验证](https://github.com/layui/layui/assets/142727418/559a0cde-497e-4bec-9ffb-6f3d1a76e881)
![提交逻辑](https://github.com/layui/layui/assets/142727418/77a5322a-50b5-40cd-806f-5c0db7336b96)

**Comment by sentsim** [comment]
如果表单是在 `<form></form>` 容器中，是否有在 form submit 事件结尾处加 `return false` ?
https://layui.dev/docs/2/form/#submit

**Reply by xiong-xiao-yang** [reply]
有的
![image](https://github.com/layui/layui/assets/142727418/9907614e-044c-4344-b7e3-b869bfd76884)

**Reply by sentsim** [reply]
那一般是前面有代码报错，导致无法阻止默认 form 提交。

**Reply by xiong-xiao-yang** [reply]
确实我自己审查就是发现会出现两次提交。我尝试了其他模块的提交，之前是没有问题的，现在也是会出现两次相同的提交，但是我明明并没有进行什么改动，前端也没有报错信息

**Reply by LSL1618** [reply]
试试在form元素上添加`onsubmit="return false;"`看看
